utils/update-next.js

```diff
@@ -1,68 +1,160 @@
+/* eslint-disable no-console */
+
+const { dirname, join } = require('path');
 const { execFileSync } = require('child_process');
+const { glob } = require('glob');
+const { pathExists, readJson, writeJson } = require('fs-extra');
```

Comment:
I dont think this will work without installing the dependencies first.

Reply:
That's easily solved. Installing the deps should only add maybe 20-30 seconds to the CI job.

Reply:
OK, forget installing deps... no need, I removed them!